apps/kyc-server/src/index.tsx

```
@@ -4,6 +4,8 @@ import { serve } from 'bun'
 import type { Server, ServerWebSocket } from 'bun'
 import { routes } from './routes'
 import { buildClient } from './utils/buildClient'
+import { createServer } from 'http'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)